src/views/Necessitous/Demand.vue

```diff
@@ -50,6 +96,22 @@ export default class NecessitousDemand extends Vue {
         return this.step();
     }
 
+    private updateSupplies(type: keyof Step.Supplies, position: any) {
+        pipe(
+            O.fromNullable(this.demand.supplies),
+            O.map(supplies => supplies[type]),
+            O.map((supply: any) => {
```

Comment:
Isn't supply infered from the record? :(

Reply:
I have errors from ts without typing